src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -254,6 +254,39 @@ public static String join(final String separator, final double[] doubles) {
     }
 
     /**
+     * Splits a String using indexOf instead of regex to speed things up.
+     *
+     * @param str the string to split.
+     * @param delimiter the delimiter used to split the string.
+     * @return an array of tokens.
+     */
+    public static ArrayList<String> split(final String str, final String delimiter) {
+        return split(str, delimiter, 10);
+    }
+
+    /**
+     * Splits a String using indexOf instead of regex to speed things up.
+     *
+     * @param str the string to split.
+     * @param delimiter the delimiter used to split the string.
+     * @param expectedNumTokens The number of tokens expected. This is used to initialize the ArrayList.
+     * @return an array of tokens.
+     */
+    public static ArrayList<String> split(final String str, final String delimiter, final int expectedNumTokens) {
```

Comment:
same comments - no `ArrayList` in return types, add argument checks and unit tests.